product_pack/README.rst

```diff
@@ -0,0 +1,129 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============
+Product Pack
+============
+
+This module extends the functionality of products to support having a
+collection of products in a single one.
+
+Product pack types
+==================
+
+This module defines the following product pack types, that will change how is
+the product shown in the sales order.
+
+Assuming *Pack* is the product pack and *subproducts* are those set to form
+the *pack*, these are your pricing options:
+
+Detailed Component Prices
+-------------------------
+
+This will show 1 line per *subproduct* and 1 line for the *pack*.
+
+All of them will use the price set in the product.
```

Comment:
Change "set in the product" by "set in the pack" or "set in the subproducts", It's not clear. By the way in takes the sum of all of them.

![2015-12-10_2-49-34](https://cloud.githubusercontent.com/assets/8736623/11704519/ba348054-9ee8-11e5-8581-d3c93baad747.png)